Tentacle/Client.swift

```diff
@@ -178,7 +178,10 @@ public final class Client {
 
         // https://developer.github.com/v3/issues/#list-issues-for-a-repository
         case IssuesInRepository(owner: String, repository: String)
-        
+
+        // https://developer.github.com/v3/issues/comments/#list-comments-on-an-issue
+        case CommentsOnIssue(number: String, owner: String, repository: String)
```

Comment:
Started using an instance of `Issue` and `Repository` here instead but it didn't feel right with the rest of the 
codebase... 🤔